companion/src/modeledit/setup.ui

```diff
@@ -537,6 +537,44 @@
          </property>
         </widget>
        </item>
+       <item row="2" column="2">
+        <widget class="QLabel" name="label_jitterFilter">
+         <property name="font">
+          <font>
+           <weight>50</weight>
+           <bold>false</bold>
+          </font>
+         </property>
+         <property name="text">
+          <string>ADC Jitter filter</string>
```

Comment:
To be consistent with radio "ADC filter". Also for layout aesthetics would prefer moving label and combobox to last grid column like Throttle trim switch

Reply:
OK, I wiIll fix it.